README.md

```diff
@@ -88,6 +89,34 @@ julia> post = sample(DynamicHMCChain, m(x=x) | (y=y,))
 (σ = 1.0±0.15, β = 0.503±0.26, α = 10.2±0.25)
 ```
 
+## How is Soss different from [Turing](https://turing.ml/dev/)?
+
+- In Soss, models are first class, and can be composed or nested. At the time of this writing, it looks like nested models will be coming soon to Turing, see [this PR](https://github.com/TuringLang/DynamicPPL.jl/pull/233).
```

Comment:
As commented on Slack, in fact it has been possible to nest models for quite some time (with the caveat that variables had to be unique), but the PR adds some convenient macro and automatic prefixing :tada:

Reply:
@devmotion is it correct to say that Turing models can be nested, but they're not first class? Does Turing have any kind of model composition?

Reply:
I would consider it first class since the design of Turing makes it possible to nest models automatically: executing a model is just calling a regular Julia function, basically as defined by the user with four additional arguments. You can use any Julia code (including macros) in the model, and hence, of course, you can also run another Turing model. Unfortunately, as large parts of Turing and the ecosystem, it's not documented currently (hopefully this will improve now that the tutorials were move from Jupyter notebooks to jmd and the webpage was refactored), but since there is no additional maintenance or development needed for nested models, I don't think they will become unsupported.